nx-dev/nx-dev/redirect-rules.js

```diff
@@ -437,6 +437,13 @@ const nxCloudUrls = {
     '/nx-cloud/recipes/on-premise/auth-saml-managed',
   '/nx-cloud/private-cloud/advanced-config':
     '/nx-cloud/recipes/on-premise/advanced-config',
+  '/concepts/dte': '/nx-cloud/concepts/dte',
+  '/nx-cloud/intro/nx-cloud-workflows': '/nx-cloud/concepts/nx-cloud-workflows',
```

Comment:
I think it should be `/nx-cloud/features/nx-cloud-workflows`.

Reply:
Good catch.  Fixed.